scout/constants/file_types.py

```diff
@@ -1,21 +1,90 @@
 # Collect general information about the file types used in Scout
+# Load priority determines load order, with lowest value loaded first.
 
 FILE_TYPE_MAP = {
-    "vcf_cancer": {"category": "cancer", "variant_type": "clinical"},
-    "vcf_cancer_sv": {"category": "cancer_sv", "variant_type": "clinical"},
-    "vcf_cancer_research": {"category": "cancer", "variant_type": "research"},
-    "vcf_cancer_sv_research": {"category": "cancer_sv", "variant_type": "research"},
-    "vcf_fusion": {"category": "fusion", "variant_type": "clinical"},
-    "vcf_fusion_research": {"category": "fusion", "variant_type": "research"},
-    "vcf_snv": {"category": "snv", "variant_type": "clinical"},
-    "vcf_snv_mt": {"category": "snv", "variant_type": "clinical"},
-    "vcf_snv_research": {"category": "snv", "variant_type": "research"},
-    "vcf_snv_research_mt": {"category": "snv", "variant_type": "research"},
-    "vcf_sv": {"category": "sv", "variant_type": "clinical"},
-    "vcf_sv_mt": {"category": "sv", "variant_type": "clinical"},
-    "vcf_sv_research": {"category": "sv", "variant_type": "research"},
-    "vcf_sv_research_mt": {"category": "sv", "variant_type": "research"},
-    "vcf_str": {"category": "str", "variant_type": "clinical"},
-    "vcf_mei": {"category": "mei", "variant_type": "clinical"},
-    "vcf_mei_research": {"category": "mei", "variant_type": "research"},
+    "vcf_cancer": {
+        "category": "cancer",
+        "variant_type": "clinical",
+        "load_priority": 10,
```

Comment:
I'm not following the reasoning behind the numbering here, why 10, 120.. and not 1,2,3 etc?

Reply:
To be able to quickly add intermediate priorities without having to renumber all.

Reply:
Mmm I'd rather change all numbers when we add a category so we have a clear sequence of numbers then having gaps in between. I find it more clear.